easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.4.1-foss-2017b.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+# Modified by: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+# Modified by: Kurt Lust, UAntwerp
+
+name = 'Bowtie2'
+version = '2.3.4.1'
+
+homepage = 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'
+description = """ Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
+ to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
+ of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
+ Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
+ its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True, 'cstd': 'c++11'}
+
+source_urls = [('http://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download')]
+sources = ['%(namelower)s-%(version)s-source.zip']
+checksums = ['a1efef603b91ecc11cfdb822087ae00ecf2dd922e03c85eea1ed7f8230c119dc']
+
+dependencies = [('tbb', '2018_U2')]
```

Comment:
@murfreesboro - is there a specific reason why you removed the `zlib` dependency?

Reply:
Okay, according to the manual it seems zlib is needed for the Bowtie2. However this file is modified from the Bowtie2-2.3.4.1-intel-2017b.eb where there is no zlib dependency. I will updated the dependencies section to include the zlib.